test/WebSites/SecurityWebSite/Controllers/AntiforgeryController.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNetCore.Mvc;
+
+namespace AjaxAntiForgeryValidation.Controllers
+{
+    [AutoValidateAntiforgeryToken]
```

Comment:
Is the inverse supported `IgnoreAntiforgeryToken` on the controller and `Validate` on the action supported? Do we need a test for that?

Reply:
I will add one in 2.1. This PR is just cherry picking a fix that was supposed to be merged in 1.0.4 and got missed.